thesis-fe/src/components/container/CalendarDashboard.js

```diff
@@ -1,24 +1,188 @@
 import React, { Component } from 'react';
 import styled from 'react-emotion'
-import plus from '../../assets/svg/plus.svg';
+import back from '../../assets/svg/back.svg';
+import { DateRange } from 'react-date-range';
+import { List } from '../container/List';
+
+const Container = styled('div')`
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #ff7e5f;  /* fallback for old browsers */
+  background: -webkit-linear-gradient(to right, #feb47b, #ff7e5f);  /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(to right, #feb47b, #ff7e5f); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+`
+
+const ContainerUsers = styled('div')`
+  margin-top: 5rem;
+  margin-left: 5rem;
+  width: 80vw;
+  height: 10vh;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  margin-bottom: 2rem;
+`
+const ContainerList = styled('div')`
+  max-height: 30rem;
+  width: 30rem;
+`
+
+const GoBackButton = styled('button')`
+  position: absolute;
+  right: 40vw;
+  margin-top: 2rem;
+  margin-right: .25rem;
+  position: relative;
+  font-size: 2rem;
+`;
+
+const H1 = styled('h1')`
+  font-size: 1.5rem;
+  margin: 1rem;
+  color: white;
+`;
+
+const H2 = styled('h1')`
+  font-size: 1.25rem;
+  margin: 1rem;
+  color: white;
+`;
 
 
 class MyTripsDashboard extends Component {
 
-  redirectToTrip = (id) => {
-    return () => {
-      this.props.history.push('/tripdetails/' + id)
+  state = {
+    tripID : this.props.location.pathname.split('/')[2],
```

Comment:
Is there a way to get the params without spliting? like this.props.match.params.id